jprov/server/proofs.go

```diff
@@ -169,6 +170,15 @@ func postProofs(cmd *cobra.Command, db *leveldb.DB, q *queue.UploadQueue, ctx *u
 	}
 
 	for {
+		interval := intervalFromCMD
+
+		if interval < 120 { // If the provider picked an interval that's less than 2 minutes, we generate a random interval for them anyways
```

Comment:
2 minutes is a pretty small window, this'll most likely still cause conflicts. I personally have my provider set to 45 minutes. Generating an interval of say 20-40 minutes seems like a more beneficial arrangement.